plugin/evm/vm_test.go

```diff
@@ -3909,7 +3909,7 @@ func TestParentBeaconRootBlock(t *testing.T) {
 				nil,
 				nil,
 				new(trie.Trie),
-				ethBlock.ExtData(),
+				*types.BlockExtras(ethBlock).Extdata,
```

Comment:
Could we add a function like `types.BlockExtData(ethBlock) []byte`?
It seems we could keep the nil check and `*types.BlockExtras(ethBlock).Extdata` seems repeated a few times.

Reply:
Done, also with `types.BlockVersion`, and also unexported both extra fields `Version` and `ExtData` since they should not need to be exported anyway with those exported functions on the block.